Awful.apk/src/main/java/com/ferg/awfulapp/ProgressListener.kt

```diff
@@ -0,0 +1,16 @@
+package com.ferg.awfulapp
```

Comment:
Does this need to be in its own file? It's only used by ``AwfulRequest`` to talk to the AwfulFragments, feels like it makes more sense to stay in there